app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+
+import EmptyNotesListImage from "images/EmptyNotesList";
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import EmptyState from "components/Common/EmptyState";
+
+import { MOCK_CONTACTS } from "./constants";
+import SideMenu from "./ContactsMenu";
+import Table from "./Table";
+import { createMockArray } from "./utils";
+
+const contacts = createMockArray(200, MOCK_CONTACTS);
+
+const Contacts = () => {
+  const [showMenu, setShowMenu] = useState(true);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  return (
+    <>
+      <SideMenu showMenu={showMenu} />
+      <Container>
+        <Header
+          title="All Contacts"
+          menuBarToggle={() => setShowMenu(!showMenu)}
```

Comment:
Please carefully read the following: https://www.bigbinary.com/react-best-practices/know-when-to-use-previous-state-for-state-update

Reply:
will do